android/sound.cpp

```diff
@@ -235,11 +235,9 @@ oboe::DataCallbackResult CSound::onAudioReady(oboe::AudioStream *oboeStream, voi
                 {
                     // copy sample received from server into output buffer
 
-
-                    // convert to 32 bit
-                    const int32_t iCurSam = static_cast<int32_t> (
-                        pSound->vecsTmpAudioSndCrdStereo [frmNum * oboeStream->getChannelCount() + channelNum] );
-                    floatData[frmNum * oboeStream->getChannelCount() + channelNum] = (float) iCurSam/ _MAXSHORT;
+                    const float fCurSam =
```

Comment:
Have you compiled and run the Android code to check that it works with your modifications? Or did you just change the code and have not compiled or tested it?

Reply:
No, I haven't check Android. Need some help here.

Reply:
Unfortunately, I cannot compile Jamulus for Android. The Android development was done mainly by Simon. But as far as I know, he still had issues on the Android platform. So maybe we can live with the fact that the changes in the Android interface are very small. So I think it is ok to take your changes untested.